extensions/resteasy-jaxb/deployment/src/main/java/io/quarkus/resteasy/jaxb/deployment/ResteasyJaxbProcessor.java

```diff
@@ -1,14 +1,54 @@
 package io.quarkus.resteasy.jaxb.deployment;
 
+import java.lang.annotation.Annotation;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.jboss.jandex.DotName;
+import org.jboss.jandex.IndexView;
+import org.jboss.resteasy.annotations.providers.jaxb.Wrapped;
+import org.jboss.resteasy.annotations.providers.jaxb.WrappedMap;
+
 import io.quarkus.deployment.Feature;
 import io.quarkus.deployment.annotations.BuildProducer;
 import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
 import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.ReflectiveClassBuildItem;
 
 public class ResteasyJaxbProcessor {
 
+    private static final List<Class<? extends Annotation>> RESTEASY_JAXB_ANNOTATIONS = Arrays.asList(
+            Wrapped.class,
+            WrappedMap.class);
+
+    @Inject
+    BuildProducer<ReflectiveClassBuildItem> reflectiveClass;
```

Comment:
Also, could you make it a parameter of the build step rather than injecting it here? It helps keeping track of the dependencies.

Reply:
Ok, will change that if it's a best practice.